backend/api/EventHandlers/InspectionFindingEventHandler.cs

```diff
@@ -103,55 +106,125 @@ private async Task<List<Finding>> GenerateFindingsList(List<InspectionFinding> i
+                cardBody.AppendLine("\"columns\": [");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"type\": \"Column\",");
+                cardBody.AppendLine($"\"items\": [");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"type\": \"TextBlock\",");
+                cardBody.AppendLine($"\"size\": \"Small\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"text\": \"{i + 1}. {findingsReports[i].TagId}\",");
+                cardBody.AppendLine("\"wrap\": true,");
+                cardBody.AppendLine($"\"style\": \"heading\"");
+                cardBody.AppendLine("},");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine("\"type\": \"FactSet\",");
+                cardBody.AppendLine("\"spacing\": \"Large\",");
+                cardBody.AppendLine("\"facts\": [");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"title\": \"Anlegg\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"value\": \"{findingsReports[i].PlantName}\"");
+                cardBody.AppendLine("},");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"title\": \"Område\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"value\": \"{findingsReports[i].AreaName}\"");
+                cardBody.AppendLine("},");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"title\": \"Tag Number\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"value\": \"{findingsReports[i].TagId}\"");
+                cardBody.AppendLine("},");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"title\": \"Beskrivelse\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"value\": \"{findingsReports[i].FindingDescription}\"");
+                cardBody.AppendLine("},");
+                cardBody.AppendLine("{");
+                cardBody.AppendLine($"\"title\": \"Tidspunkt\",");
+                cardBody.AppendLine(CultureInfo.InvariantCulture, $"\"value\": \"{findingsReports[i].Timestamp}\"");
+                cardBody.AppendLine("}");
+                cardBody.AppendLine("]");
+                cardBody.AppendLine("}");
+                cardBody.AppendLine("],");
+                cardBody.AppendLine($"\"width\": \"auto\"");
+                cardBody.AppendLine("}");
+                cardBody.AppendLine("]");
+                cardBody.AppendLine("},");
+
             }
-            return reportBuilder.ToString();
-        }
 
-        public static string GenerateAdaptiveCard(string messageContent, ILogger<InspectionFindingEventHandler> logger)
-        {
-            logger.LogInformation("Message content for Adaptive card: {message}", messageContent);
-            string adaptiveCardJson = $@"{{
```

Comment:
This old way seems more readable to me, what is the reason we can not do it this way?

Reply:
Agreed will add now.